packages/api/src/command/medical/admin/redrive-dlq.ts

```diff
@@ -74,12 +74,27 @@ async function redriveSQSUnique({
       messages.push(...messagesOfRequest);
     })
   );
+  const failed = res.flatMap(r => (r.status === "rejected" ? String(r.reason) : []));
+  if (failed.length) {
+    const succeeded = res.filter(r => r.status === "fulfilled");
+    const uniqueMsgs = uniq(failed);
+    console.log(
+      `>>> Failed to get messages from SQS (total errors ${failed.length}, ` +
+        `unique errors ${uniqueMsgs.length}, succeeded ${succeeded.length}): ` +
+        `${uniqueMsgs.join("; ")}`
```

Comment:
just confirming you want a space after the `;`?

Reply:
yes, make it easier to read the diff error messages